apps/api/src/app/bridge/bridge.module.ts

```diff
@@ -8,6 +8,7 @@ import {
   DeleteWorkflowUseCase,
   GetPreferences,
   GetWorkflowByIdsUseCase,
+  GetWorkflowWithPreferencesUseCase,
```

Comment:
when we use the `GetWorkflowByIdsUseCase` we don't always need to pull the preferences details so I decided to refactor it and create the `GetWorkflowWithPreferencesUseCase` class that pulls that information and use it in those specific places